airmozilla/base/templates/500.html

```diff
@@ -4,16 +4,29 @@
     <meta content="text/html; charset=UTF-8" http-equiv="Content-Type">
     <title>Something went wrong!</title>
     <style type="text/css">
-      body {
-        font-family: "Open Sans Light","Lucida Sans","Lucida Grande","Lucida Sans Unicode",Verdana,sans-serif;
-        background-color: rgb(32, 39, 42);
-        color: rgb(255, 255, 255);
-      }
+      body {font-family: "Open Sans Light","Lucida Sans","Lucida Grande","Lucida Sans Unicode",Verdana,sans-serif;
+        background-color:rgb(32, 39, 42);}
+      h1 {color:white}
+      h2 {color:white}
+      h3 {color:red}
+      p {color:white}
```

Comment:
Instead of white, use the same color of fonts as we do on the regular pages. I think it's `#B4B4B4`.